definitions/infra-kubernetes_statefulset/definition.yml

```diff
@@ -0,0 +1,26 @@
+domain: INFRA
+type: KUBERNETES_STATEFULSET
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+
+synthesis:
+  identifier: entity.id
+  name: k8s.statefulsetName
+  encodeIdentifierInGUID: false
+  conditions:
+  - attribute: k8s.statefulsetName
```

Comment:
Now it's failing because of this, you need to have exactly one condition 🙇 (since you already have `name: k8s.statefulsetName` you don't need that as a condition btw)

Reply:
Oh, good to know!
The plural is kind of misleading. I understood that more than one condition could be declared and wanted to take advantage of that just for extra safety. However, it's not needed at all so we're good to go 😄

Reply:
😅 yeah, it totally is misleading, sorry about that 🙇‍♀️